readthedocs/telemetry/collectors.py

```diff
@@ -151,11 +193,21 @@ def _get_pip_packages(self, include_all=False):
                 }
             ]
         """
-        cmd = ["python", "-m", "pip", "list", "--pre", "--local", "--format", "json"]
-        if not include_all:
-            cmd.append("--not-required")
+        cmd = [
+            "python",
+            "-m",
+            "pip",
+            "list",
+            "--pre",
+            "--local",
+            "--format",
+            "json",
+            "--not-required",
+        ]
```

Comment:
Seems like it would be nicer to just do a single string split on spaces?

Reply:
I don't have a strong preference on this. This chunk of code was just black-ified in this PR. However, we are using a list for all the other commands in multiple files, so I think I will keep it like that just for consistency.